src/pages/Auth/SignupPage/hooks/useSignup.ts

```diff
@@ -15,9 +15,15 @@ const useSignup = (options: ISignupOption) => {
       options.onSuccess?.();
       showToast('success', '아우누리 이메일로 인증 메일을 발송했습니다. 확인 부탁드립니다.');
     },
-    onError: (error: AxiosError<{ message?: string }>) => {
-      if (error?.response?.data) {
-        showToast('error', error.response.data.message || '에러가 발생했습니다.');
+    onError: (error) => {
+      if (isKoinError(error)) {
+        if (error.status === 409) {
+          showToast('error', '이미 가입된 이메일입니다.');
+        } else {
+          const { message } = error;
+          showToast('error', message || '회원가입에 실패했습니다.');
+          sendClientError(error);
+        }
       }
```

Comment:
KoinError가 아닌 경우를 감지하지 못하는 것 같아요~
```suggestion
    onError: (error) => {
      if (isKoinError(error)) {
        if (error.status === 409) {
          showToast('error', '이미 가입된 이메일입니다.');
          return;
        }
        else {
          const { message } = error;
          showToast('error', message || '회원가입에 실패했습니다.');
          sendClientError(error);
          return;
      }
      showToast('error', message || '회원가입에 실패했습니다.');}
      sendClientError(error);
    }
```

Reply:
`sendClientError`가 KoinError일 경우만 생각했네요! 수정 완료했습니다! a837eb33be25a027830cb801390d9be5cfd6259a